server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -2279,6 +2283,7 @@ public boolean expunge(UserVmVO vm, long callerUserId, Account caller) {
                     return false;
                 }
 
+                unassignBackupOffering(vm.getId());
```

Comment:
@Pearl1594 possible to call remove / unassign backup from backup manager here? unassign offering code will be at single place, easy to maintain.